src/frontend/src/lib/i18n/de.json

```
@@ -462,6 +462,7 @@
 			"no_infura_provider": "Kein Infura-Anbieter für Netzwerk $network",
 			"no_infura_cketh_provider": "Kein Infura CkETH-Anbieter für Netzwerk $network",
 			"no_infura_erc20_provider": "Kein Infura-ERC20-Anbieter für Netzwerk $network",
+			"no_infura_erc721_provider": "",
```

I am pretty sure you can fill this, no?  😉

```suggestion
			"no_infura_erc721_provider": "Kein Infura-ERC721-Anbieter für Netzwerk $network",
```